config/config.go

```diff
@@ -53,6 +53,8 @@ type Config struct {
 		AOFFile                string        `mapstructure:"aoffile"`
 		PersistenceEnabled     bool          `mapstructure:"persistenceenabled"`
 		WriteAOFOnCleanup      bool          `mapstructure:"writeaofoncleanup"`
+		LogLevel               string        `mapstructure:"loglevel"`
+		PrettyPrintLogs        bool          `mapstructure:"prettyprintlogs"`
```

Comment:
I think rather than using 'PrettyPrintLogs' we should use env based log ( i.e dev/prod ) allow pretty logs in dev

Reply:
+1, we can have a customisation based on envs `dev/prod` etc. 
Can be taken up as part of separate PR.

Reply:
do we already have a env variable for dev/prod? i just tried to utlise what existed